### ISSUE TITLE
Tree chop issue

### ISSUE THREAD

**Issue by Proxwian** [PandaDap2006/Pandas-Falling-Trees#50]
You can chop a large amount a trees at once if you connect them with logs

**Comment by PandaDap2006** [comment]
Yes this issue is noticed and I'm trying to update the algorithm to fix this issue.

**Comment by hammertater** [comment]
Hey @PandaDap2006, if this is an issue/exploit with TreeChop's detection, I don't mind fixing it from my end

**Comment by PandaDap2006** [comment]
Hey @hammertater, this issue has been a problem for a while now so this is on my end, support for TreeChop has also first been released today, i appreciate that you want to help.

**Comment by hammertater** [comment]
Oh wait, from the “tree chop” in the title I thought this was related to my mod, but it probably isn’t. Others have recently mentioned a similar issue for TreeChop so that’s where my brain went. My bad, carry on 😂

**Comment by Quizer9O8** [comment]
Has this bug been resolved yet?
